leeseulah/likelion-crud/src/main/java/com/likelion/likelioncrud/member/api/dto/MemberController.java

```diff
@@ -0,0 +1,62 @@
+package com.likelion.likelioncrud.member.api.dto;
+
+import com.likelion.likelioncrud.common.error.SuccessCode;
+import com.likelion.likelioncrud.common.template.ApiResTemplate;
+import com.likelion.likelioncrud.member.api.dto.request.MemberSaveRequestDto;
+import com.likelion.likelioncrud.member.api.dto.request.MemberUpdateRequestDto;
+import com.likelion.likelioncrud.member.api.dto.response.MemberInfoResponseDto;
+import com.likelion.likelioncrud.member.api.dto.response.MemberListResponseDto;
+import com.likelion.likelioncrud.member.application.MemberService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/member")
+public class MemberController {
+
+    private final MemberService memberService;
+
+    // 사용자 저장
+    @PostMapping("/save")
+    public ApiResTemplate<String> memberSave(@RequestBody MemberSaveRequestDto memberSaveRequestDto) {
+        memberService.memberSave(memberSaveRequestDto);
+        return ApiResTemplate.successWithNoContent(SuccessCode.MEMBER_SAVE_SUCCESS);
+    }
+
+    // 사용자 전체 조회
+    @GetMapping("/all")
+    public ApiResTemplate<MemberListResponseDto> memberFindAll() {
+        MemberListResponseDto memberListResponseDto = memberService.memberFindAll();
+        return ApiResTemplate.successResponse(SuccessCode.GET_SUCCESS, memberListResponseDto);
+    }
+
+    // 회원 id를 통해 특정 사용자 조회
+    @GetMapping("/{memberId}")
+    public ApiResTemplate<MemberInfoResponseDto> memberFindOne(@PathVariable("memberId") Long memberId) {
+        MemberInfoResponseDto memberInfoResponseDto = memberService.memberFindOne(memberId);
+        return ApiResTemplate.successResponse(SuccessCode.GET_SUCCESS, memberInfoResponseDto);
+    }
+    @PatchMapping ("/{memberId}")
+    public ResponseEntity<String> memberUpdate(
+            @PathVariable("memberId") Long memberId,
+            @RequestBody MemberUpdateRequestDto memberUpdateRequestDto) {
+        memberService.memberUpdate(memberId, memberUpdateRequestDto);
+        return new ResponseEntity<>("사용자 수정", HttpStatus.OK);
+
+
+    }
+    public ResponseEntity<String> memberDlete(
+            @PathVariable("memberId") Long memberId)
+            {
+        memberService.memberDelete(memberId);
+        return new ResponseEntity<>("사용자 삭제", HttpStatus.OK);
+
+
+    }
+
+
+
+}
```

Comment:
전체적으로 불필요한 공백이 많고, 62번 줄 아래에 빨간색 마이너스 표시 보이시죠 ?!
이 마이너스 표시는 파일 끝에 개행이 없을 시 깃허브에서 경고 표시를 보여주는 것입니다. 

개행이 없어도 문제는 없으나 해당 표시가 뜨지 않도록 파일 끝에 개행을 추가해 줍시다 !! 
다른 파일에도 적용되는 코멘트입니다 ☺️

[파일 끝 개행](https://hyeon9mak.github.io/github-no-newline-at-a-end-of-file/)

Reply:
나와있는대로 설정을 하긴햇는데 잘 적용됐는진 모르겟네요 ㅎㅎ ,,

Reply:
57번 줄 아래에 엔터로 개행을 추가해 주면 됩니다 !
```suggestion
}

``` 
요렇게 공백을 추가해 줍니다 ~~